### PR TITLE
Intro edits and other structuring

### PR DESCRIPTION
**Pull request by jlevy**
Bunch of small fixes following #102 and #99 

Made a variety of edits on intro to be precise about key points and reduce its length slightly. (It may still be too long.)

Other structural reworking and internal links of early sections esp around seeking professional advice.

I've also added the news emoji, which we need for tax or other recent developments.

Minor restructuring in tax subsections and more internal links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,81 +5,84 @@
...
+As of **2014** in the United States, **7.2%** of all private sector employees (**~8.5M**
 Americans) and **13.1%** of *all* employees of companies with stock held stock options.
 This number has steadily declined since **2002** levels of **12.3%** and **23.2%** respectively
 ([NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf) of General
 Social Survey data).
 
-Nearly all approaches to equity compensation are based on the idea that a ownership in a
-company will be more valuable in the future due to growth.
+Generally, equity compensation is based on the idea that ownership in a company will be
+more valuable in the future due to growth.
```
Comment (jlevy) [inline README.md]: We have to be careful about assumptions around whether an audience is are interested in startups or large companies. Ideally covering both.

File: README.md [hunk 2]
```diff
@@ -5,81 +5,84 @@
...
-exceptional companies like Apple, Microsoft, Amazon, Google, and Facebook.
-
-🌪 Controversially, public companies offer executives equity compensation in order to make
-use of a tax loophole.
-In 1993, President Bill Clinton attempted to limit executive pay with a new section of the
-Internal Revenue Code, 162(m).
+sum. And large, growing companies find offering compensation linked to ownership is more
+attractive than cash to many candidates.
+
+With the hope for growth, however, also comes risk.
```
Comment (jlevy) [inline README.md]: Risk is a different point. It likely needs a new subsection.

File: README.md [hunk 3]
```diff
@@ -5,81 +5,84 @@
...
-Of all these methods,
-[**equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp), a kind of
-ownership in the company in exchange for work, is unique in its ability to align company
-and employee interests over a long period of time, and potential for wealth creation.
-The first ~3,000 employees of Facebook split roughly $23 billion,
-[according to The Financial Times](https://www.ft.com/content/d6599ae0-5738-11e1-869b-00144feabdc0).
-
-Equity compensation is also unique in its complexity, and includes various forms, such as
-restricted stock, stock options, and RSUs.
+Some employers offer big salaries and bonuses, hip offices, work-from-home time, or
```
Comment (jlevy) [inline README.md]: Notes (@RacheltheEditor you can help me enforce these):
- Think we should be very judicious about the first links in a guide as someone will inevitably click them. Moved a formal definition of equity compensation (and that link) later.
- This FT link is paywalled so downgraded it from an inline link to a parenthetical (have added a separate issue to track that)
- The intro needs more forward links throughout to upcoming sections (added a few, could be more)

Comment (RacheltheEditor) [reply README.md]: Forward links will be easy to add in after structure is stabilized. Excellent first point. To second point, paywalled content in parenthetical is definitely better and we have to remember to add 💰emoji to those, and use them at a minimum.

**Comment by jlevy** [comment]
@RacheltheEditor I've now incorporated your last intro suggestion as well as a bunch of other reworkings of the early material. Take a look and give me your thoughts!

**Comment by jlevy** [comment]
@AndySparks @Hopeopolis feedback welcome too.

File: README.md [hunk 4]
```diff
@@ -645,34 +653,54 @@ The purpose of equity compensation is threefold:
...
+IPO? Define lockup period.
+
+🄳 The ability to buy and sell stock is called **liquidity**. In startups and many private
+companies, it is often hard to sell stock until the company is sold or IPOs, so there is
+little or no liquidity for shareholders until those events occur.
+
+🄳 A sale or IPO is often called an **exit** or **liquidity event**. Sales, dissolutions, and
+bankruptcy are all called **liquidations**.
+
+🄳 A [**dividend**](http://www.investopedia.com/terms/d/dividend.asp) is a distribution of a
```
Comment (RacheltheEditor) [inline README.md]: Brief explanation of why a company would choose to pay dividends if shareholders(in public companies) are able to make money from their shares? A loyalty program?

Comment (RacheltheEditor) [reply README.md]: (Not clear to me why dividends are in this section in any case)

File: README.md [hunk 5]
```diff
@@ -1464,15 +1428,102 @@ stock awards")).
   So, if you want to immediately exercise an option and file a Section 83(b) election, and
   you might have liquidity soon, it’s better if you can have it be an NSO.
 
+🚧 Although we refer to “restricted stock awards” in this section and once in 83b, so
+clarify that choice as well.
+We refer to “restricted stock” many times but its defining occurrence does not include a
+secondary referent of “restricted stock award.”
+
+### The AMT trap
```
Comment (RacheltheEditor) [inline README.md]: We have some more specifics (re: time of year you exercise ISOs in relation to triggering AMT and when it's too late to do anything about it) in Dummies notes. I'd like to flesh out this section a bit more! The why and wherefore of AMTs is also interesting, and the history of it helped me understand some of the details. J I'd suggest also a tax expert on this but I can add some more coverage in for the moment.

**Comment by jlevy** [comment]
👍

File: README.md (hunk 5 above)
Comment (jlevy) [reply README.md]: Sure. An AMT-only PR just dealing with AMT, with a little history (with a link?) in the tax section and any focused details only would be helpful (separate from any other language/style PR) and then we can get review.

**Pull request merged**